File: c++/maior_posicao.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {

    int n, maior, posicaoMaior;

    cout << "Quanto numeros voce vai digitar? ";
    cin >> n;

    double vet[n];

    for(int i = 0; i < n; i++) {
        cout << "Digite um numero: ";
        cin >> vet[i];
    }

    maior = vet[0];
    posicaoMaior = 0;
    for(int i = 0; i < n; i++) {
        if(vet[i] > maior) {
            maior = vet[i];
            posicaoMaior = i;
        }
    }

    cout << endl << "MAIOR VALOR = " << maior << endl;
    cout << "POSICAO DO MAIOR VALOR = " << posicaoMaior << endl;

    return 0;
}

File: java/maior_posicao.java
import java.util.Locale;
import java.util.Scanner;

public class maior_posicao {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n, posicao;
		double maior;
		
		System.out.print("Quanto numeros voce vai digitar? ");
		n = sc.nextInt();
		
		double[] vetor = new double[n];
		
		for (int i = 0; i < n; i++) {
			System.out.print("Digite um numero: ");
			vetor[i] = sc.nextDouble();
		}
		
		maior = vetor[0];
		posicao = 0;
		for (int i = 1; i < n; i++) {
			if(vetor[i] > maior) {
				maior = vetor[i];
				posicao = i;
			}
		}
		
		System.out.println();
		System.out.println("MAIOR VALOR = " + String.format("%.1f", maior));
		System.out.println("POSICAO DO MAIOR VALOR = " + posicao);	
		
		sc.close();
	}

}
